docs/change-log/index.md

```diff
@@ -1,14 +1,95 @@
+
+
+### UI
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-2811 | New Batch Inference wizard. Tech Preview.|
+| ML-2815 | The new Batch Run wizard replaces the previous New job. |
+| ML-3584 | The Model Endpoints page now displays the Function Tag. |
+| ML-4066 | The Online types list of the Target Store now includes Redis. | 
+| ML-4167 | Supports downloading the .yaml file from the Projects page. |
+| ML-4571 | The Model Endpoints page now displays the drift threshold and the drift actual value. |
+|ML-4756 | The Recents list in Jobs and Workflows (Projects pane) now displays the last 48 hours. |
+
+### Documentation
+| ID    | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3763 | Add description of configuring number of workers per GPU. See updated [Number of workers/GPUs](../runtimes/configuring-job-resources.html#number-of-workers-gpus). |
+| ML-4420 | Add configuration of memory in Spark Operator. See [Spark Operator runtime](../runtimes/spark-operator.html). |  
+| ML-2380 | Add details of V3IO and Spark runtime. See [Spark Operator runtime](../runtimes/spark-operator.html) and [Spark3Runtime](../api/mlrun.runtimes.html#mlrun.runtimes.Spark3Runtime).
+
+### Breaking changes
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3823 | The default format of `list projects` returns project names only. You can either get names or projects (`name_only`) and do a `get` only on the specific project you want (preferable), or get the full list (`full`).  [View in Git](https://github.com/mlrun/mlrun/pull/4198).|
+| ML-4171 | The Redis target implementation changed. Features-sets that use Redis as online targets must be recreated. [View in Git](https://github.com/mlrun/storey/pull/449).|
+| ML-4366 | The MLRun images `mlrun/ml-models` and `mlrun/ml-models-gpu` were deprecated and removed. The new image mlrun/mlrun-gpu is added. Additional dependencies must be installed on an as-need basis. See [MLRun images](../runtimes/images.html#mlrun-images). |
+
+
+
+### Deprecations
+
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-4075  | Python 3.7 is not supported.|
+| NA | [APIs removed in v1.5.0](##apis-removed-in-v1-5-0-deprecated-in-v1-3-0) and [CLIs removed in v1.5.0](##cli-removed-in-v1-5-0-deprecated-in-v1-3-0).  |
+| ML-4042 | [APIs deprecated in v1.5.0](##apis-deprecated-in-v1-5-0-removed-in-v1-6-0) and [CLIs deprecated in v1.5.0](##cli-deprecated-in-v1-5-0-removed-in-v1-6-0).  | 
```

Comment:
I don't see these titles were they added before?
Usually we give 2 minor versions before we remove so the removal should be in 1.7.0 but IIRC there was a special case that we forgot to deprecate and we decided to remove in 1.6.0.

Reply:
@alonmr That is what I remember also

Reply:
@alonmr I changed the layout of the deprecations, and now fixed the links.
See https://github.com/mlrun/mlrun/pull/4247